lib/oban/migrations.ex

```diff
@@ -85,14 +99,19 @@ defmodule Oban.Migrations do
   Run migrations in an alternate prefix:
 
       Oban.Migrations.up(prefix: "payments")
+
+  Run migrations in an alternate prefix but don't try to create the schema:
+
+      Oban.Migrations.up(prefix: "payments", create_schema: false)
   """
   def up(opts \\ []) when is_list(opts) do
     prefix = Keyword.get(opts, :prefix, @default_prefix)
     version = Keyword.get(opts, :version, @current_version)
+    create_schema = Keyword.get(opts, :create_schema, true)
```

Comment:
What about using `prefix != "public"` as the default here? That avoids the compound logic and special-casing in the first migration.

Reply:
You mean it would avoid the need for this if statement? (https://github.com/sorentwo/oban/pull/533/files#diff-4f2f4029f6bc09bf30159a9909a4edb2d56401527dcff6245b130f24cfc0eb71R7)
I don't think it would, because you could still have `prefix != "public"` and `create_schema = false`. That's why we are making this change, because we want to disable schema creation even when `prefix != "public"`.

Reply:
I’m suggesting that as the default for `Keyword.get`. Explicitly setting `create_schema` would override it still because it doesn’t work like an OR.

Reply:
Yes I got that. But I was wondering what you mean by "That avoids the compound logic and special-casing in the first migration.". The logic of the first migration won't change

Reply:
As is, the first migration checks both the prefix and whether a create_schema override exists. It doesn’t need to know about the special case of the public schema at all if instead it only checks whether create_schema is set.

Reply:
Ah! Now it's clear 👍  thanks. Good idea